hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerManagerImpl.java

```diff
@@ -183,6 +194,37 @@ public int getContainerStateCount(final LifeCycleState state) {
     return containerStateManager.getContainerIDs(state).size();
   }
 
+  @Override
```

Comment:
`ReplicationManager#getContainerReplicaCount` uses `legacyReplicationManager#getContainerReplicaCount` to  get ContainerReplicaCount for ratis container . legacy RM will be removed ultimately, so `legacyReplicationManager#getContainerReplicaCount` will not exist. 

i suggest to add it to ContainerManager interface for now, since it is a general interface that we can get container related info without considering the container type. after removing legacy RM, we can refactor it if needed, and  i think it will not be too much work